backend/app/analysis/aggregated_analyses.py

```diff
@@ -0,0 +1,131 @@
+"""
```

Comment:
Since this module is WIP and not wired into anything, I'm leaving this alone for now: we can finish up in follow-up work.